dinov2/data/datasets/extended.py

```diff
@@ -3,10 +3,14 @@
 # This source code is licensed under the Apache License, Version 2.0
 # found in the LICENSE file in the root directory of this source tree.
 
+from typing import Union
```

Comment:
merge lines 6 and 8: `from typing import import Any, Tuple. Union`

Reply:
Done

Reply:
Use Ruff imports to harmonize all files